src/core/public/chrome/chrome_service.tsx

```diff
@@ -116,14 +117,21 @@ export class ChromeService {
     const helpSupportUrl$ = new BehaviorSubject<string>(KIBANA_ASK_ELASTIC_LINK);
     const isNavDrawerLocked$ = new BehaviorSubject(localStorage.getItem(IS_LOCKED_KEY) === 'true');
 
+    const formatKbnVersionHeader = () => {
+      const unformattedVersion = this.params.packageInfoVersion;
+      // we assume that the version is valid
+      return `kibana-version-${unformattedVersion.split('.').join('-')}`;
```

Comment:
Is this format somehow imposed for what we're going to do with it? If not, I would follow our css naming convention and go with

`kbnVersion--${unformattedVersion.split('.').join('-')}`

Reply:
I'm not sure if the format is imposed. @joshdover how much room do we have to rather use `kbnVersion-....`?

Reply:
I went ahead and [changed](https://github.com/elastic/kibana/pull/102443/files#diff-f586b4a13b87b4d9192b54dcd897a24b881f9308a401f4422319c3ffff36044eR128) it to `kbnVersion-${...}`.